src/main/java/edu/iu/uits/mail/MailSigner.java

```diff
@@ -94,6 +103,21 @@ public MailSigner(Properties properties) {
         }
     }
 
+    public String getEmailPassword(String alias) {
+        String emailKeyPassword = properties.getProperty(String.format(CERT_PASSWORD_PROPERTY_TEMPLATE, alias));
+        if (emailKeyPassword == null || emailKeyPassword.trim().isEmpty()) {
+            // try local part of address
+            log.debug(String.format("No key password found for %s.  Trying local portion, %s.", alias, alias.replaceAll(LOCAL_ADDRESS_REGEX, "$1")));
+            emailKeyPassword = properties.getProperty(String.format("mail.keystore.%s.password", alias.replaceAll(LOCAL_ADDRESS_REGEX, "$1")));
+            if (emailKeyPassword == null || emailKeyPassword.trim().isEmpty()) {
+                // default to keystore password
+                log.debug(String.format("No key password found for %s.  Defaulting to using the keystore password.", alias));
```

Comment:
I think your use of `alias` here would cause it to print the full email instead of just the local part?

Reply:
This issue has still not been addressed

Reply:
Sorry, I misunderstood this as there was another minor issue I fixed. I had intended that message to indicate a general failure of finding the sender address in any form. I'm updating the message to include both the full email and the local part.